getting_started.md

```diff
@@ -18,28 +18,34 @@ or
 2. [Lite](https://fpgasoftware.intel.com/20.1/?edition=lite&platform=windows) - Cyclone V for the Audio Mini
 
 ### Windows Subsystem for Linux
-1. Install [WLS](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with Ubuntu 18.04 LTS or newer 
+1. Install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with Ubuntu 18.04 LTS or newer 
 
 ### Python 3.8.X
 1. In windows command prompt, run `python --version`
 2. Install Python 3.8 from the Microsoft Store
 
 ## Configure the FrOST Autogen Software
-1. In WSL, clone the [Docs Repo](https://github.com/fpga-open-speech-tools/docs) into the `[FrOST Repos]` Directory 
-    - `sudo git clone https://github.com/fpga-open-speech-tools/docs.git`
-    - Note: The `Frost_Autogen_Config.sh` script will fail if the Docs repo is cloned using GitHub Desktop on Windows. GitHub Desktop converts Unix EOL characters to Dos EOL characters. Use the following steps to convert the Unix EOL Characters to Dos EOL Characters. 
+1. Create a new folder to store all of the FrOST Repos. This folder will be refered to as the `[FrOST Repos]` Directory in the rest of this guide.
+2. In WSL:  
```

Comment:
Change steps 2-3 from being run in WSL to being run from the command prompt with the command `wsl.ex`
This will fix the issue of not installing on the default WSL Distro